pyart/util/datetime_utils.py

```diff
@@ -1,30 +1,62 @@
 """ Functions for converting date and time between various forms """
 
-from netCDF4 import num2date
+from netCDF4 import num2date, date2num
 
+EPOCH_UNITS = "seconds since 1970-01-01T00:00:00Z"
 
-def datetime_from_radar(radar):
+def datetime_from_radar(radar, epoch=False):
```

Comment:
There needs to be two blank lines before this function to meet PEP8 conventions.